### ISSUE TITLE
residential landuses without inside node with place=* tag

### ISSUE THREAD

**Issue by severinmenard** [osm-fr/osmose-backend#1261]
I did not find a analyzer to identify landuse=residential without any inside node with place=* tag. It may be unnecessary for towns and cities but it is good to smallest residential landuse e.g. in rural areas, as the place tag provides an information regarding the number of dwelling (place=isolated_dwelling or place=hamlet) so we are checking team eg. in Africa. Easy to do with QGIS, but would be great to have it as well in Osmose.

**Comment by frodrigo** [comment]
Unsure about this.
On detailed area polygon landuse may be small and away, while part of a bigger place.

Size may be a criteria along distance to other landuse places.

Note, place is not always on a node and may be on directly on the landuse.

**Comment by severinmenard** [comment]
I agree for large polygons, this is why I provided use cases related to small residential landuse in rural areas, great if a size criteria can be added.
Indeed for place tags on landuse, this is actually unfortunate IMO and anyway a minority use (less than 400K occurrences over 5.5M on node according to https://taginfo.openstreetmap.org/keys/place#combinations). Osmose users are used to untick any unwanted analyze showing false positive for specific tag scheme or specific quality checks.

**Comment by Bibi56** [comment]
When the limits are well known it's better to tag as a relation, see https://www.openstreetmap.org/relation/10060749 for instance.
For dwellings, it's quite usual not to have a residential area around it. https://www.openstreetmap.org/node/3366835425 is such an example.

**Comment by severinmenard** [comment]
Thank you for your examples in France, but they do not fit at all with the situation in Africa
